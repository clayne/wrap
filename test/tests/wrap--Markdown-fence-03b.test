wrap | /dev/null | -u | md-code-fence-03b.md | 0
