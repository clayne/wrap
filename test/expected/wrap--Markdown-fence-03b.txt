This is a line of text.  This is another line of text.
~~~
int main( void ) {
  printf( "hello, world\n" );
}
~~~~
This is a third line of text
but it should be treated as code
because the closing fence
doesn't have the same number of characters
as the opening fence.
